GovUkHtmlHelperExtensions.cs

```diff
@@ -373,26 +375,6 @@ public static class GovUkHtmlHelperExtensions
                 textInputAppendix);
         }
 
-        public static IHtmlContent GovUkTextInputFor<TModel>(

```

Comment:
Is this deliberate?

Reply:
Nope, caused by a rebase by  me I think, will fix now